### DISCUSSION TITLE
Regarding Validator to check the variable accepts number or string.(Not an alphanumeric)

### DISCUSSION THREAD

**Discussion by mohanraj-rr** [adonisjs/core#4475] [Help]
how to check the variable accepts number or string, but this variable is optional too. And i like to add rules in it.

**Comment by RomainLanz** [comment]
Hey @mohanraj-rr! 👋🏻 

What have you tried so far?
Please, look at [vine documentation](https://vinejs.dev/docs/schema_101) to learn more on how to use our validator.

**Reply by mohanraj-rr** [reply]
Hi @RomainLanz!

**Reply by mohanraj-rr** [reply]
If you don't mind, specify it for adonis v5
